lib/tarantool_server.py

```diff
@@ -833,12 +843,12 @@ def cleanup(self, *args, **kwargs):
     def start(self, silent=True, wait=True, wait_load=True, rais=True, args=[],
               **kwargs):
         if self._start_against_running:
-            return
+            return True
```

Comment:
Is it some artifact from the old version of the patch? AFAIU, the return value is not used anywhere.

Reply:
Fixed.